server/service/src/processors/transfer/shipment/test.rs

```diff
@@ -578,7 +578,7 @@ impl ShipmentTransferTester {
         );
         assert_eq!(inbound_shipment.colour, None);
         assert_eq!(inbound_shipment.user_id, None);
-        assert_eq!(inbound_shipment.on_hold, false);
+        assert!(!inbound_shipment.on_hold);
```

Comment:
There's quite a few of these changes. I'm personally find the old version easier to read but I'm ok with this if we want it, also happy to disable this lint...

Reply:
Both same readability to me... can ask team~

Reply:
Find the first version better as well but I am ok going with the clippy suggestion